javascript-stringify.js

```diff
@@ -229,6 +229,11 @@
       return PRIMITIVE_TYPES[typeof value](value, indent, next);
     }
 
+    // Handle Error objects.
+    if (value instanceof Error) {
```

Comment:
BTW, about [the issue I mentioned](https://github.com/alexkuz/redux-devtools-inspector/issues/42), is it expected that `stringify` can return not only a string but also `undefined`?